src/graphql-server/dnaInterfaces/HhaDnaInterface.js

```diff
@@ -50,6 +54,34 @@ const HhaDnaInterface = {
         happStoreId,
         isEnabled: true
       })))
+  },
+
+  hostPricing: {
+    get: async () => {
+      // we need an id to call get_service_log_details, and because we set all apps the same in add_service_log_details, it doesn't matter which app the id comes from
+      const allAvailable = await HhaDnaInterface.happs.allAvailable()
+      if (isEmpty(allAvailable)) throw new Error("Can't set Host Pricing: no happs available to host.")
+      return createZomeCall('provider/get_service_log_details')({ app_hash: allAvailable[0].id })
```

Comment:
yeah, I've been thinking about that. Reason I haven't moved them into constants yet is because we mostly (/ever?) don't use them multiple times, so the dnaInterface files *are* the source of truth for the zome calls. I could go either way for now, and if and when we start needing them in multiple places, then they should go in constants.

A more ripe candidate is the routing paths. Just haven't got around to putting them in a file yet